apps/site/src/views/Home/sections/Mentor/Mentor.module.scss

```diff
@@ -0,0 +1,132 @@
+@use "zothacks-theme" as theme;
+@use "bootstrap-utils" as bootstrap;
+
+.applySticky {
+	@include bootstrap.rfs(400px, height);
+	@include bootstrap.rfs(15%, margin-left);
+	background-image: url("~@/assets/images/MentorStickyYellow.svg");
+	background-repeat: no-repeat;
+	background-size: contain;
+	padding-top: 3rem;
+	padding-left: 5rem;
+	padding-right: 5rem;
+	padding-bottom: 8rem;
+	position: relative;
+	z-index: 1;
+
+	h2 {
+		@include bootstrap.font-size(2rem);
+		@include bootstrap.padding(2rem);
+	}
+
+	a {
+		@include bootstrap.font-size(1.25rem);
+		position: absolute;
+		bottom: 10%;
+		left: 6%;
+		transform: rotate(7.5deg);
+	}
+
+	p {
+		display: none;
+	}
+
+	@include bootstrap.media-breakpoint-down(lg) {
+		background-image: none;
+		background-color: theme.$sticky-yellow;
+		background-size: contain;
+		margin-left: 0;
+		height: auto;
+		padding: 0;
+		box-shadow: 0 3px 5px 3px lightgray;
+
+		.applyStickyContent {
+			@include bootstrap.padding(3rem);
+
+			h2 {
+				padding: 0;
+			}
+		}
+
+		a {
+			display: none;
+		}
+
+		p {
+			display: block;
+		}
+	}
+}
+
+.descSticky {
+	background-image: url("~@/assets/images/MentorStickyGreen.svg");
+	background-repeat: no-repeat;
+	background-size: contain;
+	background-position: 60%;
+	padding: 6rem;
+	position: absolute;
+	z-index: 0;
+	top: 15%;
+	left: 40%;
+	width: 41.667%;
+
+	p {
+		@include bootstrap.padding(1rem);
+		transform: rotate(-5deg);
+	}
+
+	a {
+		display: none;
+	}
+
+	@include bootstrap.media-breakpoint-down(xxl) {
+		top: 10%;
+		left: 47.5%;
+	}
+
+	@include bootstrap.media-breakpoint-down(xl) {
+		width: 50%;
+		padding: 5rem;
+	}
+
+	@include bootstrap.media-breakpoint-down(lg) {
```

Comment:
Note: Bootstrap suggests a "Mobile first" design approach
> Bootstrap’s CSS aims to apply the bare minimum of styles to make a layout work at the smallest breakpoint, and then layers on styles to adjust that design for larger devices ([Bootstrap Breakpoints, Core Concepts](https://getbootstrap.com/docs/5.3/layout/breakpoints/#core-concepts))

As such, using `media-breakpoint-up` would be more preferred than `media-breakpoint-down`. However, given the complexity of the design, this is not a huge deal, but something to keep in mind for future implementations.